plugins/nf-wave/src/main/io/seqera/wave/plugin/WaveClient.groovy

```diff
@@ -57,6 +57,8 @@ class WaveClient {
 
     private static Logger log = LoggerFactory.getLogger(WaveClient)
 
+    private static final List<String> DEFAULT_CONDA_CHANNELS = ['defaults', 'conda-forge']
```

Comment:
The default should be `['conda-forge', 'defaults']`. The channel `conda-forge` should have priority over `defaults`. Is the normal way to use it.

For example if you run:
```
conda create -n test -c defaults -c conda-forge numpy
 ```
  it will install `numpy=1.23.4` from the `defaults` channel. While if you run
```
conda create -n test -c conda-forge -c defaults numpy
```
it will install `numpy=1.23.5` from `conda-forge` channel.

Reply:
I remember they advice to use `conda-forge` over `defaults`. See here https://conda-forge.org/docs/user/tipsandtricks.html#how-to-fix-it

Reply:
Yes, that's exactly what I mean. To use `conda-forge` over `defaults` the default list of channels should be `['conda-forge', 'defaults']` and not `['defaults', 'conda-forge']`.

Reply:
Damn, I was convinced I had written the other way around 😄